src/setup_payload/SetupPayload.cpp

```diff
@@ -32,14 +32,16 @@
 
 namespace chip {
 
-bool IsCHIPTag(uint8_t tag)
+// Spec 5.1.4.2 CHIPCommon tag numbers are in the range [0x00, 0x7F]
+bool IsCommonTag(uint8_t tag)
 {
-    return tag >= (1 << kRawVendorTagLengthInBits);
+    return tag < (1 << kRawVendorTagLengthInBits);
```

Comment:
```suggestion
    return tag < 0x80;
```
I don't think `kRawVendorTagLengthInBits` makes sense in this new setup as a concept.